test/test-functions

```diff
@@ -96,8 +97,12 @@ $KERNEL_APPEND \
 }
 
 run_nspawn() {
+    local _nspawn_cmd="../../systemd-nspawn --register=no --kill-signal=SIGKILL --directory=$TESTDIR/nspawn-root $ROOTLIBDIR/systemd $KERNEL_APPEND"
```

Comment:
I agree.
But that [`Makefile`](https://github.com/systemd/systemd/blob/master/test/TEST-01-BASIC/Makefile) doesn't contain/pass to scripts `autoconf`'s variables.

I think we should create a `test-functions.in` with:
```
local _nspawn_cmd="@builddir@/systemd-nspawn ..."
```
Also we can change:
```
if ! ROOTLIBDIR=$(pkg-config --variable=systemdutildir systemd); then
    echo "WARNING! Cannot determine rootlibdir from pkg-config, assuming /usr/lib/systemd" >&2
    ROOTLIBDIR=/usr/lib/systemd
fi
```
to
```
@systemdutildir@
```
and so on.

But this is another patch.